### ISSUE TITLE
Dot should be invalid in Latitude and Longitude in manual location 

### ISSUE THREAD

**Issue by shiv07tiwari** [sky-map-team/stardroid#232]
**Summary:** 

When a user enters manual location , only putting a dot in latitude and longitude shall not be accepted. 
**Steps to reproduce:** 

Setings -> Location -> Manual

**Comment by jaydeetay** [comment]
Right- it looks like we need better validation - at the moment we seem to accept the values and then flash up a toast later.  I think it's not worth doing anything fancy in V1.0 but V2.0 needs to have a much slicker experience.
